README.md

```diff
@@ -44,10 +46,11 @@ I wrote a "[blog post](/claims/blog_post.md)" that claimed I was "mourning" my b
 Because I used Strava, a popular fitness tracking program, a [local blogger](https://sfcitizen.com/blog/2012/04/06/did-strava-com-help-kill-pedestrian-sutchi-hui-whats-your-time-on-the-castro-street-descent-aka-castro-street-bomb/), assumed I was racing at the time of the accident | Using Strava to record all my rides does not mean that I was racing that day or that I ever engage in real or imaginary races while riding on crowded city streets; rather, it means I like data
 I ran multiple red lights and stop signs leading up to the crash, according to one eyewitness | The witness report, which surfaced more than a week after the accident, sounded spurious, but I had no way to disprove it
-I rode a brakeless fixed-gear bicycle (or "fixie") | The police report contains several photos of the bike, held in their custody, which has drop bars and brifters, 18 gears (2x9), a freewheel and fully-functioning front and rear brakes
+I rode a brakeless fixed-gear bicycle (or "fixie") | The police report contained several photos of the bike, held in their custody, which had drop bars and brifters, 18 gears (2x9), a freewheel and fully-functioning front and rear brakes
 I was riding recklessly | No one driving a car would ever be called "reckless" for entering an intersection on a yellow signal in the high 20s or low 30s in a 25mph zone
-I hit the pedestrian "dead-on," I was "hunched down" and that there was "never a moment" where I was "trying to slow down." | I knew I had braked hard and swerved, and thought I had tried to crash my bike to avoid pedestrians, but again, had no way to prove it
+I hit the pedestrian "dead-on," I was "hunched down" and there was "never a moment" where I was "trying to slow down." | I knew I had braked hard and swerved, and thought I had tried to crash my bike to avoid pedestrians, but again, had no way to prove it
 [A tracker on Bucchere’s bike allegedly showed he was riding faster than 35 mph in a 25-mph zone](https://www.sfgate.com/bayarea/matier-ross/article/S-F-bicyclist-in-fatal-crash-may-face-felony-3507830.php) | There was no "tracker;" this estimate was based on Strava's wildly-inaccurate interpolation of GPS waypoints collected by my iPhone
 I gave statements to the police that didn't match the surveillance video, including saying "the crosswalk was filled with people coming from both directions" but "the video shows only three or four people in the crosswalk when the collision occurred." | I remembered at least twice as many, but had no way to prove it—at least, not yet
+The DA denied leaking information to the local press and/or blamed the SFPD | Dean Taylor, an inspector in the SFPD department that investigated the accident, told me the DA's office stole the accident video from a locked filing cabinet and leaked false information about it to the local press, saying the DA's behavior was "not fair" and that my case was "complete bullshit"
```

Comment:
an inspector in the SFPD [remove the word "department"] [replace "that" with "who] investigated the accident ...